docs/examples/irf_tool_config.json

```diff
@@ -42,6 +42,6 @@
     "bkg_fov_offset_n_edges": 21,
     "source_offset_min": 0.0001,
     "source_offset_max": 1.0001,
```

Comment:
These should be changed to 0.001 and 1.001 to reflect the 100 bins

Reply:
Why start at a value > 0?

Reply:
I think I had some confusion when I first started with the values and did not correct them, as I had not received any comments about them.
I will change them
```
source_offset_min = 0
source_offset_max = 1
source_offset_n_edges = 101
```
That should be fine for default right?

Reply:
They will at least create 100 evenly spaced bins between 0 and 1 deg which sounds reasonable.